diagram.md

```diff
@@ -191,28 +193,39 @@ The `Cluster Elements` contain all of the video frames and audio for each track.
 ```
 Figure: Representation of a `Cluster Element` and its immediate `Child Elements`.
 
-Below is a representation of the Block structure.
+```
++---------------------------------------------+
+| Block | Portion of | Data Type   | Bit Flag |
+|       | a Block    |             |          |
+|       |-------------------------------------+
+|       | Header     | TrackNumber |
+|       |            |-------------|
+|       |            | Timecode    |
+|       |            |------------------------+
```

Comment:
I like diagrams, but I agree that the "bit flag" column makes this a bit ambiguous. We could combine both approaches:

* Leave the diagram in but remove the "bit flag" column
* Describe the flags in a sentence or two below the diagram